File: LEVEL 2/repeating_char.py
def find_repeating_character(string):
    input_str = string.upper()
    char_count = {}

    for char in input_str:
        if char.isalpha():
            char_count[char] = char_count.get(char, 0) + 1

    repeat = False

    for char, count in char_count.items():
        if count > 1:
            print(f"{char}{count} times repeated")
            repeat = True

    if not repeat:
        print("No repeated characters in string")

user_input = input()
find_repeating_character(user_input)

File: LEVEL 2/repeating_char.c
#include<stdio.h>
#include<string.h>
#include<ctype.h>

void main() {
    char string[1000];
    scanf("%[^\n]s", string);

    int count[256] = {0};

    for (int i=0; i<strlen(string); i++) {
        char curChar = string[i];

        curChar = toupper(curChar);

        if (isalpha(curChar)) {
            count[curChar]++;
        }
    }

    int repeat = 0;
    for (int i=0; i<256; i++) {
        if (count[i] > 1) {
            printf("%c%d times repeated\n", i, count[i]);
            repeat=1;
        }
    }

    if (!repeat) {
        printf("No repeated characters in string\n");
    }
}